MobileCenter/MobileCenter/Internals/Application Delegate/MSAppDelegateForwarder.m

```diff
@@ -215,30 +240,85 @@ - (BOOL)application:(UIApplication *)app
   return [[MSAppDelegateForwarder sharedInstance] application:app openURL:url options:options returnedValue:result];
 }
 
+- (void)custom_application:(UIApplication *)app didRegisterForRemoteNotificationsWithDeviceToken:(NSData *)deviceToken {
```

Comment:
The implementation is repeating. Probably you can take it as a private method and then call it from other methods.